07-Publishing-Repositories.Rmd

```diff
@@ -4,7 +4,46 @@ editor_options:
     wrap: 80
 ---
 
-# Publishing Repositories
+# Archiving Data and Code
```

Comment:
To do still: add a section about data bias, and describing it in the metadata. After our work stoppage we landed on incorporating the following: 

Use of [the concepts introduced in Libby Barnes' datasheets](https://github.com/dmhuehol/Datasheets-for-Earth-Science-Datasets/tree/main) to document known biases, but also think critically about biases in general. We make decisions as we clean and collate data, and this introduces biases; these should be clearly documented.

Also a blurb about responsible AI use: [See also this article](https://www.cambridge.org/core/journals/environmental-data-science/article/why-we-need-to-focus-on-developing-ethical-responsible-and-trustworthy-artificial-intelligence-approaches-for-environmental-science/8A32BECCE5FBB518F982855FAB6553D4) about the importance of this in AI/ML in geosciences, however these principles START at data handling/publishing.

Reply:
Thank you for reminding me of this - it was further down in my list of 'to-do's' and I missed it. Adding it now.

Reply:
resolved with recent push - new rendered .html in teams.